app/uk/gov/hmrc/customs/notification/controllers/CustomsNotificationController.scala

```diff
@@ -17,8 +17,8 @@
 package uk.gov.hmrc.customs.notification.controllers
 
 import java.util.UUID
-import javax.inject.{Inject, Singleton}
 
```

Comment:
The controller needs to wait until the notification is either saved in DB or sent to Pull-Q